c++/src/kj/exception.c++

```diff
@@ -609,13 +611,16 @@ namespace {
                          "\nstack: ", stringifyStackTraceAddresses(trace),
                          stringifyStackTrace(trace), '\n');
 
+#ifndef KJ_NO_POSIX
```

Comment:
Removing the last two lines of the crash handler seems to make it useless. Should the entire `crashHandler()` function be `#if`'d out instead? It looks like it won't be used anyway.